### DISCUSSION TITLE
Stats API is missing time_on_page metrics?

### DISCUSSION THREAD

**Discussion by bfritscher** [plausible/analytics#2727] [Anything Else]
Is it possible to get time_on_page via the api? The information is on the dashboard, but not in the stats api. Is there a way to compute it from the current available information in the api?

**Comment by bogplau** [comment]
Hi there, 

You're correct about the time on page missing from the API. Currently only visit duration is available via the API. The list of available metrics is available [here](https://plausible.io/docs/stats-api#metrics). 

We updated threads when new features are released, but please also send us an email to hello@plausible.io with this feature request so we can I can follow up as soon as we'll add this to the APIs. 

Thank you!
